Mathlib/Computability/DFA.lean

```diff
@@ -91,11 +92,19 @@ theorem evalFrom_of_append (start : σ) (x y : List α) :
   x.foldl_append _ _ y
 #align DFA.eval_from_of_append DFA.evalFrom_of_append
 
+/--
+`M.acceptsFrom start` is the language of `x` such that `M.evalFrom start x` is an accept state.
+-/
+def acceptsFrom (start : σ) : Language α := {x | M.evalFrom start x ∈ M.accept}
+
+theorem mem_acceptsFrom (start : σ) (x : List α) :
+    x ∈ M.acceptsFrom start ↔ M.evalFrom start x ∈ M.accept := by rfl
```

Comment:
```suggestion
theorem mem_acceptsFrom {start : σ} {x : List α} :
    x ∈ M.acceptsFrom start ↔ M.evalFrom start x ∈ M.accept := by rfl
```
Btw should that lemma be simp?

Reply:
As [discussed on Zulip](https://leanprover.zulipchat.com/#narrow/stream/144837-PR-reviews/topic/.2311372.20DFA.2EacceptsFrom.2C.20DFA.2Emap.2C.20DFA.2Eequiv/near/427126311), I added some notes on this to the header docs.